src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -160,13 +160,25 @@ object AddDatasetMetadataToDeposit {
     )
   }
 
+  def createRelations(dataset: Dataset) = {
+    dataset.rowsWithValuesFor(composedRelationFields).map { row =>
```

Comment:
`rowsWithValuesFor` is een relatief dure operatie. Kan deze methode niet op een andere manier worden geïmplementeerd?